packages/app-builder-lib/src/util/yarn.ts

```diff
@@ -148,15 +146,15 @@ export interface RebuildOptions {
 }
 
 /** @internal */
-export async function rebuild(appDir: string, buildFromSource: boolean, frameworkInfo: DesktopFrameworkInfo, arch = process.arch) {
+export async function rebuild(appDir: string, buildFromSource: boolean, frameworkInfo: DesktopFrameworkInfo, arch: Arch) {
   log.info({ arch, version: frameworkInfo.version, appDir }, "executing @electron/rebuild")
   const rootPath = await searchModule.getProjectRootPath(appDir)
   const options: electronRebuild.RebuildOptions = {
     buildPath: appDir,
     electronVersion: frameworkInfo.version,
-    arch,
-    debug: log.isDebugEnabled,
+    arch: Arch[arch],
     projectRootPath: rootPath,
```

Comment:
can you also add a different `cachePath` argument for each arch? 

https://github.com/electron/rebuild/blob/main/src/types.ts#L10

Reply:
What for? Why can't the default cache be used?